modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -186,18 +186,32 @@ class NDB_Form_Imaging_Browser extends NDB_Form
                 }
             }
 
+            // Provide a default pic for DTI with time dimention / volumes
+            // processed before 2012-07, after this date, they are fixed.
+            $config = NDB_Config::singleton();
+            $check_pic_filename = $FileObj->getParameter('check_pic_filename');
+            $Time = number_format($FileObj->getParameter('time'), 2);
+            if (!file_exists(
+                $config->getSetting('imagePath') . "pic/" .
+                $check_pic_filename
+            ) && $Time > 0) {
+                $check_pic_filename = str_replace(
+                    '_check.jpg',
+                    '_check-0.jpg',
```

Comment:
@gluneau 

Do you think this fix is standard across projects? in other words, do we know for sure if the DTI file has a time dimension and the jpg is not displayed in imaging browser, then we should be looking for the file with -0 addition before .jpg?
I think this is a very specific solution to the problem you encountered, I am just curious if we can assume it is a Loris-wide change.

@cmadjar,
can you please check with your data as you are likely to have DTI scans prior to 2012.